engine/pkg/externalresource/internal/file_manager.go

```diff
@@ -77,6 +77,9 @@ type FileManager interface {
 	// GetPersistedResource returns the descriptor of an already persisted resource.
 	GetPersistedResource(ctx context.Context, ident ResourceIdent) (ResourceDescriptor, error)
 
+	// CleanOrRecreatePersistedResource cleans the persisted resource.
+	CleanOrRecreatePersistedResource(ctx context.Context, ident ResourceIdent) (ResourceDescriptor, error)
```

Comment:
`CleanOrRecreatePersistedResource` sounds like it's doing either cleaning or recreating, but it actually does both.

Maybe just `RecreatePersistedResource` is enough.

Reply:
For local filemanager, it simply removes the resource and recreates it. 
For s3 filemanager, it either [cleans files](https://github.com/pingcap/tiflow/pull/7491/files#diff-1c2eff62ba26b13484fc11619e06322ed96dd751247d7fd0523946030aacbe37R157) or [rebuilds resources](https://github.com/pingcap/tiflow/pull/7491/files#diff-1c2eff62ba26b13484fc11619e06322ed96dd751247d7fd0523946030aacbe37R141).

Reply:
Please update the comment on this method.